plugins/table.md

```diff
@@ -83,6 +83,30 @@ tinymce.init({
 });
 ```
 
+### `table_use_colgroups`
+
+{{ site.requires_5_5v }}
+
+This option adds `colgroup` and `col` elements to new tables for specifying column widths. Existing tables and tables added using `setContent` or paste are not affected.
+
+{{site.productname}} only supports the `width` attribute on `col` elements. Other attributes are not supported, such as the `span` attribute.
+
+**Type:** `Boolean`
+
+**Default Value:** `false`
+
+**Possible Values:**  `true`, `false`
+
+##### Example
```

Comment:
Should this be a h4 instead of a h5, and with a more descriptive title?

Reply:
```suggestion
#### Example: Enabling `colgroup` support using `table_use_colgroups`
```